.travis.yml

```diff
@@ -1,30 +0,0 @@
-language: python
-matrix:
-  include:
-    - python: 2.7
-      dist: trusty
-    - python: 3.4
-      dist: trusty
-    - python: 3.5
-      dist: trusty
-    - python: 3.6
-      dist: trusty
-    - python: 3.7
-      dist: xenial
-    - python: 3.8
-      dist: xenial
-
-before_install:
-  - if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
-      echo "No pull requests can be sent to the master branch" 1>&2;
-      exit 1;
```

Comment:
Can we convert this into a GitHub action? This was a helpful check to have as it is really easy to miss if a pull request is targeting the master branch.

Reply:
Currently it wont even run tests if its made against non-develop branch. Do we want to run them and then error out?

Reply:
I was thinking we just have a separate action that just fails out on submitters to tell them we do not accept PR's sent to master. The main reason to have this is so we can signal back to the submitter to switch the target branch themselves. Otherwise, it is not obvious that you messed up the PR by sending it to master.

Reply:
Offline conversation... We are just going to send this update to fail on PRs to master as a separate PR.